### PR TITLE
add the mex function for bodySpatialMotionPD

### PR DESCRIPTION
**Pull request by hongkai-dai**
I used this mex function to debug `bodySpatialMotionPD` in controlUtil.cpp

### CODE REVIEW

File: systems/controllers/CMakeLists.txt [hunk 1]
```diff
@@ -64,5 +64,9 @@ if (MATLAB_FOUND)
   include_directories (${CMAKE_SOURCE_DIR}/util )
   target_link_libraries(individualCentersOfPressuremex drakeRBM drakeUtil drakeControlUtil)
 
+  add_mex(bodySpatialMotionPDmex bodySpatialMotionPDmex.cpp)
+  include_directories (${CMAKE_SOURCE_DIR}/util )
```
Comment (RussTedrake) [inline systems/controllers/CMakeLists.txt]: don't need this again (include_directories will work for the rest of the file and children)

File: systems/controllers/bodySpatialMotionPDmex.cpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  if(mxGetM(prhs[rhs_args]) != nq || mxGetN(prhs[rhs_args]) != 1)
+  {
+    mexErrMsgTxt("Drake:bodySpatialMotionPD: q should be nq x 1");
+  }
+  Map<VectorXd> q(mxGetPr(prhs[rhs_args++]),nq);
+  if(mxGetM(prhs[rhs_args]) != nq || mxGetN(prhs[rhs_args]) != 1)
+  {
+    mexErrMsgTxt("Drake:bodySpatialMotionPD: qd should be nq x 1");
+  }
+  Map<VectorXd> qd(mxGetPr(prhs[rhs_args++]),nq);
```
Comment (RussTedrake) [inline systems/controllers/bodySpatialMotionPDmex.cpp]: mxGetPrSafe

Comment (RussTedrake) [inline systems/controllers/bodySpatialMotionPDmex.cpp]: and all instances below.

File: systems/controllers/bodySpatialMotionPDmex.cpp [hunk 3]
```diff
@@ -0,0 +1,80 @@
+#include "controlUtil.h"
+#include "drakeUtil.h"
+#include "drakeGeometryUtil.h"
+#include "mex.h"
+
+using namespace Eigen;
+using namespace std;
+
+void mexFunction(int nlhs, mxArray* plhs[], int nrhs, const mxArray* prhs[])
```
Comment (RussTedrake) [inline systems/controllers/bodySpatialMotionPDmex.cpp]: add comment describing the inputs

**Comment by RussTedrake** [comment]
otherwise looks good.

File: systems/controllers/bodySpatialMotionPDmex.cpp [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+{
+  if(nrhs != 11)
```
Comment (psiorx) [inline systems/controllers/bodySpatialMotionPDmex.cpp]: Should be a space after if before the open paren.  Also, the curly bracket should open on the same line.

File: systems/controllers/bodySpatialMotionPDmex.cpp [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  Map<Matrix<double,6,1>> Kd(mxGetPrSafe(prhs[rhs_args++]));
+  if(mxGetM(prhs[rhs_args]) != 7 || mxGetN(prhs[rhs_args]) != 1)
+  {
+    mexErrMsgTxt("Drake:bodySpatialMotionPD: xyzquat_task_to_world should be 7 x 1");
+  }
+  Map<Matrix<double,7,1>> xyzquat_task_to_world(mxGetPrSafe(prhs[rhs_args++]));
+  Isometry3d T_task_to_world;
+  T_task_to_world.translation() = xyzquat_task_to_world.head<3>();
+  T_task_to_world.linear() = quat2rotmat((xyzquat_task_to_world.tail<4>()).eval());
+  Vector6d body_vdot = bodySpatialMotionPD(r, robot_state, body_index, body_pose_des, body_v_des, body_vdot_des, Kp,Kd, T_task_to_world);
```
Comment (psiorx) [inline systems/controllers/bodySpatialMotionPDmex.cpp]: Should bodySpatialMotionPD be a member of RigidBodyManipulator?  It always takes in a reference to an RBM and operates on its internal body_indices .  

@tkoolen any comment?

Comment (hongkai-dai) [inline systems/controllers/bodySpatialMotionPDmex.cpp]: I do not think so. Conceptually bodySpatialMotionPD works for a single
body, it does not have to tie with RigidBodyManipulator.

Comment (tkoolen) [inline systems/controllers/bodySpatialMotionPDmex.cpp]: I agree with Hongkai. Also so far I don't think we have any controls related code in RBM, and I think we should keep it that way.

**Pull request merged**